### DISCUSSION TITLE
Why stop support for armv7sf-k2.6, x86-k2.6 and armv5sf-k3.2?

### DISCUSSION THREAD

**Discussion by maurerr** [Entware/Entware#1018] [Announcements]
I thought Broadcom sdk-k.26 arm routers were the biggest users of entware - out of curiosity - why drop support ?
https://github.com/Entware/Entware/commit/a8dd4452aa9446d4c399d2ee0b8dc7840f046655

thanks

**Comment by ryzhovau** [comment]
That's it. armv7sf-k2.6, x86-k2.6 and armv5sf-k3.2 feeds are EoS. No further updates. Porting new software to ancient SDK becomes harder and harder. In some cases, it draws too many attentions of our team. So we'll use our (limited) resources to bring new Rust packages and other new features.

I know how «surprised» will be those guys who use it, I'm realized it as an impact to part of SNB/Tomato/DD-WRT/<you name it» community, but we have to move one.

As usual, any package ever been created for Entware stays at bin.entware.net. I've made some [notes](https://github.com/Entware/Entware/wiki/Compile-packages-from-sources) on Wiki for those who wants to compile something new for personal needs.

**Comment by wellloaded** [comment]
The fact that it's becoming more and more difficult perhaps also means that is more and more appreciated. No disrespect meant, but do you actually have so many other systems other than router devices that might benefit from entware? Modern x86 platforms often don't use a squashfs and have full access to disks so they can install distribution packages directly (without entware).

Again I'm not after an argument here at all, I'm just genuinely curious to know the stats of what has been put EoL Vs what is still maintained. 

I might be wrong as I don't have any number to back up my statement but I have the feeling you might be losing finger-in-the-air above 60% of the current user set and it's a shame because you guys really did an amazing job for those platforms and helped resolving some tedious scenarios via the entware option.

That said this message is more to thank you for what done so far rather than criticize the EoL decision.

Thanks!

**Comment by ryzhovau** [comment]
Got it, NP! 

As for me personally, I keep this project in a constant state where I can gain max users benefits with less efforts from my side. Since establishment, we (now we are a small team) wittingly avoid resource-consuming variants of Entware development to make sure we can support it in the future. This guideline helps out not to be buried under Entware support. And that's why we can keep taken obligations for more than decade.

> I might be wrong as I don't have any number to back up my statement but I have the feeling you might be losing finger-in-the-air above 60% of the current user set and it's a shame because you guys really did an amazing job for those platforms and helped resolving some tedious scenarios via the entware option.

You can always find monthly deployment stats in [Telegram channel](https://t.me/entware), here is the last one:
```
• 83.17% of hits is mipselsf-k3.4,
• 10.68% of hits is armv7sf-k3.2,
• 2.30% of hits is x64-k3.2,
• 2.03% of hits is aarch64-k3.10,
• 0.99% of hits is armv7sf-k2.6,
• 0.30% of hits is mipssf-k3.4,
• 0.29% of hits is armv5sf-k3.2,
• 0.24% of hits is x86-k2.6,
```

Still, we stay in touch with SNB/Tomato/other communities, knowing its core members who's ready to develop k2.6 based feeds on their own. We are ready to provide them all necessary access to our infrastructure. So, now it's up to them.

**Reply by maurerr** [reply]
`Still, we stay in touch with SNB/Tomato/other communities, knowing its core members who's ready to develop k2.6 based feeds on their own. We are ready to provide them all necessary access to our infrastructure. So, now it's up to them.`
yeah I'm actually interested on taking over the maintenance of a limited number of packages for armv7sf-k2.6 (the ones I use + the ones that compiles cleanly without patches/hacks) but I'm busy atm so I'll have have more time to look into this next month

**Comment by maurerr** [comment]
my repo is available at https://maurerr.github.io/entware-armv7-k26/  and source code fork https://github.com/maurerr/Entware/tree/armv7-2.6

**Reply by haha252** [reply]
Thank you!

**Comment by ValdikSS** [comment]
@ryzhovau could you please update the wiki page stating that these repos are frozen, in the table?
https://github.com/Entware/Entware/wiki/Alternative-install-vs-standard

Update: nevermind, I have the privilege for wiki, updated the table.
